src/AddressBookGUI.java

```diff
@@ -9,25 +9,20 @@
 import java.util.regex.Pattern;
 
 public class AddressBookGUI extends JFrame {
-    //TODO change save as to not be usable
     private static final long serialVersionUID = 1L;
 
-    private static void createAndShowGUI() {
-        AddressBook addressBook = new AddressBook();
-        AddressBookController controller = new AddressBookController(addressBook);
-        AddressBookGUI gui = new AddressBookGUI(controller, addressBook);
-        gui.setVisible(true);
-    }
-
     public static void main(String[] args) throws ClassNotFoundException {
         Class.forName("org.sqlite.JDBC");
-        SwingUtilities.invokeLater(() -> createAndShowGUI());
+        SwingUtilities.invokeLater(() -> {
+            AddressBookGUI gui = new AddressBookGUI();
+            gui.setVisible(true);
+        });
     }
 
-    private final AddressBookController controller;
-    private final AddressBook addressBook;
-    private final JTable nameList;
-    private final TableRowSorter<AddressBook> tableRowSorter;
+    private final AddressBook addressBook = new AddressBook();;
+    private final AddressBookController controller = new AddressBookController(addressBook);
+    private final JTable nameList = new JTable(addressBook);
+    private final TableRowSorter<AddressBook> tableRowSorter = new TableRowSorter<>(addressBook);;
     private final JButton addButton = new JButton("Add...");
     private final JButton editButton = new JButton("Edit...");
     private final JButton deleteButton = new JButton("Delete");
```

Comment:
I recognize my code! Just leave it, it will get overwritten when I merge mine anyway.